docusaurus/docs-internals/editor-controls/multivalue-controls/margin.md

```diff
@@ -48,30 +63,26 @@ Returns an object with the following values:
 }
 ```
 
-`marginType` - Indicates if the same margin is applied to all spaces (`"grouped"`) or if each space has different margin (`"ungrouped"`); <br/>
-`margin` - The margin value applied to all spaces when `marginType` is `"grouped"`; <br/>
-`marginSuffix` - The unit for the uniform margin, either `"px"` for pixels or `"%"` for percentages; <br/>
-`marginTop` - The margin value applied specifically to the top space; <br/>
-`marginTopSuffix` - The unit for the top space margin, either `"px"` or `"%"`; <br/>
-`marginRight` - The margin value applied specifically to the right space; <br/>
-`marginRightSuffix` - The unit for the right space margin, either `"px"` or `"%"`; <br/>
-`marginBottom` - The margin value applied specifically to the bottom space; <br/>
-`marginBottomSuffix` - The unit for the bottom space margin, either `"px"` or `"%"`; <br/>
-`marginLeft` - The margin value applied specifically to the left space; <br/>
-`marginLeftSuffix` - The unit for the left space margin, either `"px"` or `"%"`; <br/>
-
-### Usage
-
-#### Basic example
-Standard definition with only the required keys. This control will be displayed on all devices.
+Example of value:
 
 ```js
 {
-  id: "margin", 
-  type: "margin"
+  marginType: "grouped",
+  margin: 10,
+  marginSuffix: "px",
+  marginTop: 0,
+  marginTopSuffix: "px",
+  marginRight: 0,
+  marginRightSuffix: "px",
+  marginBottom: 0,
+  marginBottomSuffix: "px",
+  marginLeft: 0,
+  marginLeftSuffix: "px",
```

Comment:
open the editor and make some `console.log` to check the real keys

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/08328ff0-cb49-4a5d-b34d-8d82173a9d56)

above keys from screen is keys with "margin" prefix